### DISCUSSION TITLE
Why does it take so long to start spectate in multi?

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#21922] [Q&A] [answered]
https://user-images.githubusercontent.com/86934170/209952341-aa50524a-300d-4f8d-b521-9c99fa4dd202.mp4

Is it like known issue? You should wait 20 seconds until every screen start do an action.

**Comment by peppy** [comment, marked as answer]
Yes this is definitely broken. Tracking via https://github.com/ppy/osu/issues/22091.
